extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -6,6 +6,104 @@
  */
 
 /**
+ * Restores infobox dialog info when going back from media dialog
+ *
+ * @param {ve.ui.WindowManager} win
+ * @param {ve.dm.WikiaTransclusionModel} model - original tranclusion model of infobox dialog
+ * @param {string} imageName - image name from media dialog
+ * @param {string} paramName - Name of a field (e.g. image1)
+ */
+function restoreInfoboxDialog(win, model, imageName, paramName) {
+	if (imageName) {
+		model.parts[0].params[paramName].value = imageName;
+	}
+
+	win.transclusionModel = model;
```

Comment:
Is this a temporary property that only we use? If so, I would add a comment describing why we need it.